### PR TITLE
Added doc about handling reboots

### PR DESCRIPTION
**Pull request by bobbytreed**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [ ] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[62ed2f5](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/62ed2f5a89e0ea1ddf904b0e770235d9dfbf6a1f)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[62ed2f5](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/62ed2f5a89e0ea1ddf904b0e770235d9dfbf6a1f)_: 

:clock10: **Incremental building**: average incremental building time is _13 min(s) 17 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[62ed2f5](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/62ed2f5a89e0ea1ddf904b0e770235d9dfbf6a1f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/TOC.yml](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) |
[dsc/configurations/reboot-a-node.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/configurations/reboot-a-node.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C17%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201901171137172687-3563%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=j8P%2FKcpteu%2B0%2B0k%2Fe8HebD3VDU%2Fhb%2Fk0RDeCgjSwQGs%3D&st=2019-01-17T11%3A47%3A49Z&se=2019-02-17T11%3A52%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[38fa49b](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/38fa49bbc757d2d9fc59f1979e9f104a94c04b0b)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[38fa49b](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/38fa49bbc757d2d9fc59f1979e9f104a94c04b0b)_: 

:clock10: **Incremental building**: average incremental building time is _13 min(s) 18 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[38fa49b](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/38fa49bbc757d2d9fc59f1979e9f104a94c04b0b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/TOC.yml](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) |
[dsc/configurations/reboot-a-node.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/configurations/reboot-a-node.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C17%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201901171137515428-3563%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=h4Ly1xgvf8L4Gbn9TcutH3CqUWdX%2BdkQ3pykjIXFKLg%3D&st=2019-01-17T12%3A02%3A33Z&se=2019-02-17T12%3A07%3A33Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: dsc/configurations/reboot-a-node.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+ms.date:  1/17/2019
+keywords:  dsc,powershell,configuration,setup
+title:  Reboot a Node
+---
+# Reboot a Node
+
+> [!NOTE]
+> This topic talks about how to reboot a Node. In order for the reboot to be successful the **ActionAfterReboot** and **RebootNodeIfNeeded** LCM settings need to be configured properly. To read about Local Configuration Manager settings, see [Configure the Local Configuration Manager](../managing-nodes/metaConfig.md), or [Configure the Local Configuration Manager (v4)](../managing-nodes/metaConfig4.md).
```
Comment (sdwheeler) [inline dsc/configurations/reboot-a-node.md]: Use semantic line breaks with a max line length of 100. Do this for all paragraphs.

File: dsc/configurations/reboot-a-node.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+Nodes can be rebooted from within a resource, by using the `$global:DSCMachineStatus` flag. Setting this flag to `1` in the `Set-TargetResource` function forces the LCM to reboot the Node directly after the **Set** method of the current resource. Using this flag, the **xPendingReboot** resource detects if a reboot is pending outside of DSC.
+
+Your [configurations](configurations.md) may perform steps that require the Node to reboot. This could inclue things such as:
```
Comment (sdwheeler) [inline dsc/configurations/reboot-a-node.md]: inclue ==> include

File: dsc/configurations/reboot-a-node.md (hunk 1 above)
Comment (bobbytreed) [reply dsc/configurations/reboot-a-node.md]: Complete

File: dsc/configurations/reboot-a-node.md (hunk 2 above)
Comment (bobbytreed) [reply dsc/configurations/reboot-a-node.md]: Complete

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[39dfa53](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/39dfa53682fd6af02c66ce92898076a761157658)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 11 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[39dfa53](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/39dfa53682fd6af02c66ce92898076a761157658)_: 

:clock10: **Incremental building**: average incremental building time is _13 min(s) 17 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[39dfa53](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/39dfa53682fd6af02c66ce92898076a761157658)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/TOC.yml](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) |
[dsc/configurations/reboot-a-node.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/configurations/reboot-a-node.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C17%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201901171621346236-3563%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5wgH2Mw9kFjGY8aFiCJetDjsJYkjAjDIWXbbaJftPWA%3D&st=2019-01-17T16%3A32%3A16Z&se=2019-02-17T16%3A37%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[94d3778](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/94d377806294fc2f8e432280acc9acdfa7fc82ff)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[94d3778](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/94d377806294fc2f8e432280acc9acdfa7fc82ff)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 35 sec(s)_

**Comment by mgreenegit** [comment]
@bobbytreed this doc is looking great we just need to make one change.  We need to make it clear that any DSC resource can implement LCM handling reboots by setting the machine status during Set.  xPendingReboot is only needed if you need to detect that something outside of DSC has left the machine pending reboot.

In the example given, the computer is renamed by a DSC resource that [already implements setting the machine status](https://github.com/PowerShell/ComputerManagementDsc/blob/e2c87bab88ba383547e20bbc47a0a1bd9be3a146/Modules/ComputerManagementDsc/DSCResources/MSFT_Computer/MSFT_Computer.psm1#L375).

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[94d3778](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/94d377806294fc2f8e432280acc9acdfa7fc82ff)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/TOC.yml](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) |
[dsc/configurations/reboot-a-node.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/configurations/reboot-a-node.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C24%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201901242039376390-3563%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=vSMHvETd7BYZKiXJlrx%2FB1a41xJcluJQQnD9pJnS63s%3D&st=2019-01-24T20%3A48%3A59Z&se=2019-02-24T20%3A53%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[3662726](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/36627269bd535a9ca65c9704bdc072d7ade0f8cc)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[3662726](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/36627269bd535a9ca65c9704bdc072d7ade0f8cc)_: 

:clock10: **Incremental building**: average incremental building time is _13 min(s) 2 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[3662726](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/36627269bd535a9ca65c9704bdc072d7ade0f8cc)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/configurations/reboot-a-node.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/configurations/reboot-a-node.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) | [Details](#user-content-dsc%2Fconfigurations%2Freboot-a-node.md)
[dsc/TOC.yml](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) |

<a name="dsc%2Fconfigurations%2Freboot-a-node.md"></a>
### [dsc/configurations/reboot-a-node.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/configurations/reboot-a-node.md)
 - **Line 71**: **[Warning]** Illegal link: `&lt;a href=&quot;configurations/configDataCredentials.md#Handling-Credentials-in-DSC&quot;&gt;Handling Credentials in DSC&lt;/a&gt;` -- missing bookmark. The file configurations/configDataCredentials.md doesn&#39;t contain a bookmark named Handling-Credentials-in-DSC.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C25%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201901251859456417-3563%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=LWZFPnN5OgWLXfytzU1vN5jdZ1oRCQOJB%2B0%2F5BRixYk%3D&st=2019-01-25T19%3A09%3A36Z&se=2019-02-25T19%3A14%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[dd1fd1f](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/dd1fd1f089d762e885c1d7580a58356ec94fdf63)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[dd1fd1f](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/dd1fd1f089d762e885c1d7580a58356ec94fdf63)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 59 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[dd1fd1f](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/dd1fd1f089d762e885c1d7580a58356ec94fdf63)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/configurations/reboot-a-node.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/configurations/reboot-a-node.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) | [Details](#user-content-dsc%2Fconfigurations%2Freboot-a-node.md)
[dsc/TOC.yml](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) |

<a name="dsc%2Fconfigurations%2Freboot-a-node.md"></a>
### [dsc/configurations/reboot-a-node.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/configurations/reboot-a-node.md)
 - **Line 71**: **[Warning]** Illegal link: `&lt;a href=&quot;configurations/configDataCredentials.md#Handling-Credentials-in-DSC&quot;&gt;Handling Credentials in DSC&lt;/a&gt;` -- missing bookmark. The file configurations/configDataCredentials.md doesn&#39;t contain a bookmark named Handling-Credentials-in-DSC.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C25%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201901252244045951-3563%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=fpr5oh4219DnZpmNhNrBRRDFk%2FYK7j0pSBwkvgFnY5Y%3D&st=2019-01-25T22%3A53%3A21Z&se=2019-02-25T22%3A58%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[80b2839](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/80b28396f5af9051b8ade876c2cb442ec58f0245)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[80b2839](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/80b28396f5af9051b8ade876c2cb442ec58f0245)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 59 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[80b2839](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/80b28396f5af9051b8ade876c2cb442ec58f0245)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/TOC.yml](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) |
[dsc/configurations/reboot-a-node.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/configurations/reboot-a-node.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C25%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201901252255529915-3563%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=62RtNoHUc240qpglOV6vGw9PW6ZcwEEhnZUYw6Dd3b0%3D&st=2019-01-25T23%3A07%3A01Z&se=2019-02-25T23%3A12%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[a9f1035](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/a9f1035dacf2bf96a00927708fa32e760528afac)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[a9f1035](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/a9f1035dacf2bf96a00927708fa32e760528afac)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 59 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[a9f1035](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/a9f1035dacf2bf96a00927708fa32e760528afac)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/TOC.yml](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) |
[dsc/configurations/reboot-a-node.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/configurations/reboot-a-node.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C26%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201901252349147554-3563%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=6Ddm9HYIpBhNxNy9u87g5txN80GeFR8tU96KfI%2FySFY%3D&st=2019-01-25T23%3A57%3A38Z&se=2019-02-26T00%3A02%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[c116490](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/c116490fc4c8a3152f270ec5cf5c591e0a43783d)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[c116490](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/c116490fc4c8a3152f270ec5cf5c591e0a43783d)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[c116490](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/c116490fc4c8a3152f270ec5cf5c591e0a43783d)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 54 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[c116490](https://github.com/bobbytreed/PowerShell-Docs/pull/3563/commits/c116490fc4c8a3152f270ec5cf5c591e0a43783d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/TOC.yml](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) |
[dsc/configurations/reboot-a-node.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-1429323/dsc/configurations/reboot-a-node.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/configurations/reboot-a-node?branch=pr-en-us-3563) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C28%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201901281710050609-3563%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=DAbRuGy0apQUo5yruRsgo%2BYOEK3t3gQtC8dP5zvrlyc%3D&st=2019-01-28T17%3A18%3A32Z&se=2019-02-28T17%3A23%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**